content/applications/finance/accounting/fiscal_localizations/localizations/italy_IT.rst

```diff
@@ -1,6 +1,6 @@
-==========
-Italy (IT)
-==========
+=====================
+Italy (Pec mail) (IT)
+=====================
```

Comment:
Not sure what is the best move here.

There is two italian e-invoicing system : SDICoop (web-service) that is the newly added one, in English and Pec mail (via e-mail) an old system that will eventually be removed, and the doc is in Italian.

Hence the two separate files. I think it's important to keep the different titles so that people are conscious that the two things are different. What is your opinion about file structure ?

Reply:
I would suggest doing this (+ adapt all the other headings)

```suggestion
==========
Italy (IT)
==========

.. _italy/pec:

PEC mail
========
```
And you can add a `:ref:` link in italy.rst in the warning at line 13.

As I said, ideally, all the content would be only in one single file in English (italy.rst) and the translation would be done in transifex.